bindings/nodejs/examples/21-collect-outputs.js

```diff
@@ -48,7 +48,7 @@ async function handleNewOutputOfBob(err, data) {
     if (event.accountIndex === bob.meta.index) {
         const outputId = event.event.NewOutput.output.outputId
         await bob.sync()
-        const resp = await bob.collectOutputs([outputId])
+        const resp = await bob.claimOutputs([outputId])
```

Comment:
Could you also rename the file name and the log?

Reply:
Sure!